src/main/java/io/pakland/mdas/githubstats/domain/model/HistoricQueries.java

```diff
@@ -0,0 +1,25 @@
+package io.pakland.mdas.githubstats.domain.model;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name = "historic_queries")
+public class HistoricQueries {
+    @Id
+    @Column(name = "id", updatable = false, nullable = false)
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @ManyToOne()
+    @JoinColumn(name = "team_id")
+    private Team teamId;
+
+    @Column(name = "name", nullable = false)
+    private String name;
+
+    @Column(name = "from", nullable = false)
+    private String from;
+
+    @Column(name = "to", nullable = false)
+    private String to;
+}
```

Comment:
I suppose that the Entity class doesn't have any getter/setter on purpose. Let's define it later on another  refactor issue when we know (tell don't ask), and also respect encapsulation principles (don't create getters/setters if not needed).

Reply:
Yes, exactly. I remember not seeing any other entity with methods defined in them.